package.json

```diff
@@ -313,8 +313,8 @@
     "husky": "3.0.5",
     "immer": "9.0.7",
     "inquirer": "1.2.3",
-    "jest": "^28.1.3",
-    "jest-environment-jsdom": "^28.1.3",
+    "jest": "^29.5.0",
```

Comment:
To resolve the issue outlined in [this comment](https://github.com/artsy/force/pull/12243#discussion_r1166038345) jest version was upgraded and a [new config option](https://github.com/artsy/force/pull/12243/commits/665e02d6d6a02b6662afd771db20117c84486434) is now used ([available starting v29+](https://jestjs.io/docs/configuration/#workeridlememorylimit-numberstring)) that limits jest worker memory.